### PR TITLE
Update slashing specs

### PR DESCRIPTION
**Pull request by brentstone**
These are spec updates that mirror the work done in #892.

### CODE REVIEW

File: documentation/specs/src/economics/proof-of-stake/cubic-slashing.md [hunk 1]
```diff
@@ -106,22 +102,25 @@ As a function, it can be drawn as (assuming $r_{\text{min}} = 1\%$):
...
+3. Jail (and freeze) the misbehaving validator and remove it from the validator set, effective at the beginning of the next epoch. Bonds can still be made to validators while frozen. Unbonds can be made from validators while they are jailed, but not while frozen. At the pipeline offset only, if the misbehaving validator was removed from the consensus set, replace it with the appropriate below-capacity validator.
 
-This can be implemented as a negative inflation rate for a particular block.
+Now, at the beginning of each new epoch, if there are slashes enqueued for this epoch, then:
+1. Compute the cubic slash rate for slashes in this epoch.
+For each validator that has enqueued slashes,
+2. Compute the rate for each of the validator slashes. The total rate by which this validator is slashed is the sum of these rates, capped at `1.0`. Slash the validator voting powers with the total rate, enforcing that the slashed voting power be 0 if it would otherwise become negative.
+3. Send the slashed token amount from the proof-of-stake address to the slash pool address. These can ultimately be used by governance to refund validators from wrongful slashes (among other things?).
```
Comment (cwgoes) [inline documentation/specs/src/economics/proof-of-stake/cubic-slashing.md:123]:
```suggestion
3. Send the slashed token amount from the proof-of-stake address to the slash pool address. These could ultimately be used by governance to refund validators for slashes which are determined to have been accidental, or just burned.
```

File: documentation/specs/src/economics/proof-of-stake/cubic-slashing.md [hunk 2]
```diff
@@ -106,22 +102,25 @@ As a function, it can be drawn as (assuming $r_{\text{min}} = 1\%$):
...
+3. Jail (and freeze) the misbehaving validator and remove it from the validator set, effective at the beginning of the next epoch. Bonds can still be made to validators while frozen. Unbonds can be made from validators while they are jailed, but not while frozen. At the pipeline offset only, if the misbehaving validator was removed from the consensus set, replace it with the appropriate below-capacity validator.
 
-This can be implemented as a negative inflation rate for a particular block.
+Now, at the beginning of each new epoch, if there are slashes enqueued for this epoch, then:
+1. Compute the cubic slash rate for slashes in this epoch.
+For each validator that has enqueued slashes,
+2. Compute the rate for each of the validator slashes. The total rate by which this validator is slashed is the sum of these rates, capped at `1.0`. Slash the validator voting powers with the total rate, enforcing that the slashed voting power be 0 if it would otherwise become negative.
```
Comment (cwgoes) [inline documentation/specs/src/economics/proof-of-stake/cubic-slashing.md:122]: We should link to the part of the spec that explains how slashes are lazily processed (still negative inflation, right?)

**Review by cwgoes — APPROVED** [review]
ACK modulo minor comments

File: documentation/specs/src/economics/proof-of-stake/cubic-slashing.md (hunk 2 above)
Comment (brentstone) [reply documentation/specs/src/economics/proof-of-stake/cubic-slashing.md:122]: Which part in particular are you talking about? There was previously one sentence about "negative inflation" at the bottom of this file, but that's it, so I removed it.

Comment (brentstone) [reply documentation/specs/src/economics/proof-of-stake/cubic-slashing.md:122]: In the `bonding-mechanism.md`. I'll make some updates there and link to the proper place

**Comment by cwgoes** [comment]
@brentstone Is this ready for be merged, or do you plan to make more changes?

**Comment by brentstone** [comment]
> @brentstone Is this ready for be merged, or do you plan to make more changes?

Plan to make some more changes.

**Comment by brentstone** [comment]
Closing in favor of https://github.com/anoma/namada-docs/pull/14 in the new repo.

**Pull request closed without merging**